File: tests/fixtures/usage.go

package main

import "fmt"

func greet(name string) {
	fmt.Println("Hello,", name)
}

type Greeter struct{}

func (g Greeter) Greet(name string) {
	fmt.Println("Hello,", name)
}

func main() {
	greet("Go user")

	g := Greeter{}
	g.Greet("Go user")

	message := "Hello"
	anotherMessage := message
}

File: tests/fixtures/usage.java

class Greeter {
    public static void main(String[] args) {
        System.out.println("Hello, Java!");
    }

    public void greet(String name) {
        System.out.println("Hello, " + name);
    }
}

class Main {
    public static void main(String[] args) {
        Greeter greeter = new Greeter();
        greeter.greet("Java user");

        String message = "Hello";
        String anotherMessage = message;
    }
}
